fluent-plugin-systemd.gemspec

```diff
@@ -23,6 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "test-unit", "~> 2.5"
   spec.add_development_dependency "reevoocop"
 
-  spec.add_runtime_dependency "fluentd", "~> 0.12"
+  spec.add_runtime_dependency "fluentd", "~> 0.13"
```

Comment:
In fact I don't even understand what this is for there is no fluentd 0.13 please drop this change...

Reply:
#43 changed the following line to:

```
spec.add_runtime_dependency "systemd-journal", "~> 1.3"
```

I think that is the change you want here ...

Reply:
Thanks, that's definitely the line I wanted to change....